kautomator/src/main/kotlin/com/kaspersky/components/kautomator/component/common/assertions/UiBaseAssertions.kt

```diff
@@ -26,7 +27,18 @@ interface UiBaseAssertions {
      * Checks if the view is not completely displayed
      */
     fun isNotDisplayed() {
-        assertThat(view.loadView()).isFalse()
+        // the implementation is a little bit tricky because
+        // UiObjectInteractionDelegate throws UnfoundedUiObjectException in case of absence of a related UIObject2
+        // that's why we just catch the mentioned exception and do a final check
+        val notDisplayed = try {
```

Comment:
Why did you change the previous behavior?

Reply:
The previous solution required an additional method of `UiObjectInteractionDelegate` which was too poor because made an abstraction leak.